tests/unit/insertTestResult.unitTest.ts

```diff
@@ -1357,10 +1357,10 @@ describe("insertTestResult", () => {
         });
     });
 
-    context("when inserting a testResult that has an ADR testType without expiryDate", () => {
+    context("when inserting a testResult that has a pass ADR testType without expiryDate", () => {
         it("should throw 400 and descriptive error message", () => {
             const testResultWithAdrTestTypeWithoutExpiryDate = testResultsPostMock[6];
-            delete testResultWithAdrTestTypeWithoutExpiryDate.testTypes[0].testExpiryDate;
```

Comment:
Will this scenario (no testExpiryDate entry at all, or testExpiryDate undefined) not actually occur?

Reply:
Front end will always send a field of this name - null if no real  value